src/Libraries/CoreNodeModels/CustomSelectionItem.cs

```diff
@@ -0,0 +1,165 @@
+using System;
+using System.ComponentModel;
+using System.Windows.Input;
+using Newtonsoft.Json;
+
+namespace CoreNodeModels
+{
+    /// <summary>
+    /// Represents an Enum item
+    /// </summary>
+    public class CustomSelectionItem
+    {
+        public string Name { get; set; }
```

Comment:
public properties should have `///` summary tags. These will generate compiler warnings otherwise. Also part of our standard https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards - mostly just follow MSDN standards for c#.